packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -272,6 +272,39 @@ export class EntityService {
     }
   }
 
+  private async checkServiceEntityLicense(resource: Resource) {
+    if (!this.billingService.isBillingEnabled) {
+      return;
+    }
+    await this.checkServiceLicense(resource);
```

Comment:
@overbit  Why do we need serviceCheck in the entityCheck method?

Reply:
to avoid duplicating code as it was there before